packages/devtools_app/lib/src/screens/memory/memory_heap_tree_view.dart

```diff
@@ -41,7 +41,7 @@ import 'primitives/memory_utils.dart';
 // TODO(bkonyi): enable new allocation profile table when we're ready to remove
 // the existing allocations table.
 @visibleForTesting
-bool enableNewAllocationProfileTable = false;
+bool enableNewAllocationProfileTable = true;
```

Comment:
is this ready to flip on by default?

Reply:
I can't wait to get rid of this flag so I can stop forgetting to flip it back... :-)